apps/docs/pages/guides/platform/branching.mdx

```diff
@@ -0,0 +1,526 @@
+
+<Admonition type="note">
+
+Only schema changes are captured in preview branches.
+
+Preview instances contain no data by default. You can include a seed file to seed your preview instance with sample data when the Preview Branch is created. Future versions of branching may allow for automated data seeding and cloning.
+
+</Admonition>
+
+#### Git providers
+
+To facilitate Supabase Branching, your Supabase project will be connected to a Git repository via an integration.
+
+Supabase Branching is planned to work with various Git providers, like GitHub, GitLab or BitBucket.
+
+Our first supported Git provider uses [GitHub](#branching-with-github).
+
+### Branching with GitHub
+
+Supabase Branching uses the Supabase GitHub integration to read files from your GitHub repository. With this integration, Supabase watches all commits, branches, and pull requests of your GitHub repository so that Preview Branches can be correctly created and managed.
+
+One of your Git branches will be a designated Production Branch which is will also reflect your Supabase project's Production Branch.
+
+For every Git Branch in your GitHub Repository, you may have a corresponding Preview Branch. Each time a new Preview Branch is created the migrations in the Git branch of that Preview Branch are run on the Preview Branch.
+
+The Preview Branch is also seeded with sample data based on `./supabase/seed.sql`, if that file exists.
+
+<figure className="max-w-[700px] mx-auto">
+  <Image
+    className="hidden dark:block"
+    alt="Each branch has a separate Supabase instance."
+    src="/docs/img/guides/platform/branching/github-workflow.jpg"
+  />
+  <Image
+    className="dark:hidden"
+    alt="Each branch has a separate Supabase instance."
+    src="/docs/img/guides/platform/branching/github-workflow--light.jpg"
+  />
+  <figcaption>Each branch has a separate Supabase instance.</figcaption>
+</figure>
+
+Each time a commit is pushed with new migrations in the `./supabase/migrations` directory, the migrations are run on the matching preview instance.
+
+<figure className="max-w-[700px] mx-auto">
+  <Image
+    className="hidden dark:block"
+    alt="Each GitHub branch can have its own Supabase preview branch."
+    src="/docs/img/guides/platform/branching/github-workflow-commit-migration.jpg"
+  />
+  <Image
+    className="dark:hidden"
+    alt="Each GitHub branch can have its own Supabase preview branch."
+    src="/docs/img/guides/platform/branching/github-workflow-commit-migration--light.jpg"
+  />
+  <figcaption>Each GitHub branch can have its own Supabase preview branch.</figcaption>
+</figure>
+
+Only GitHub is currently supported (as a beta release). If you’re interested in other Git providers or a non-Git-based workflow, join the discussions for _GitLab Branching_, _BitBucket Branching_, and non-Git based Branching.
+
+### Alternatives to branching
```

Comment:
```suggestion
## Alternatives to branching
```

Reply:
Are you sure, feels like we're just removing the level 2 TOCs